documentation/modules/cruise-control/con-rebalance-performance.adoc

```diff
@@ -9,6 +9,33 @@
 You can adjust several performance tuning options for cluster rebalances.
 These options control how partition replica and leadership movements in a rebalance are executed, as well as the bandwidth that is allocated to a rebalance operation.
 
+== Rebalancing modes
+
+When using the `KafkaRebalance` custom resource for running a rebalancing operation, there are three available modes: `full`, `add-brokers` and `remove-brokers`.
+The mode is specified through the corresponding `spec.mode` field in the `KafkaRebalance` custom resource.
+
+The `full` mode runs a full rebalancing by moving replicas across all the brokers in the cluster.
+This is the default mode if the `spec.mode` field is not defined in the `KafkaRebalance` custom resource.
+
+The `add-brokers` mode can be used after scaling up the Apache Kafka cluster by adding one or more brokers.
+After a scaling up, the new brokers are used to host only partitions of new created topics.
+It means that the newly added brokers are actually not used even for long time if no new topics are created.
```

Comment:
"This means that, if no new topics are created, the newly added brokers are not used and existing broker remain under the same load."